### PR TITLE
[OU-IMP] apriori : pos_order_line_no_unlink features has been merged into Point of Sale. 

### PR DESCRIPTION
**Pull request by legalsylvain**
See related PR : https://github.com/OCA/pos/pull/989

cc reviewer of the previous PR : @enriquemartin, @rafaelbn, @carmenbianca, @FrancoMaxime

### CODE REVIEW

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -79,6 +79,8 @@
     "coupon_commercial_partner_applicability": "loyalty_partner_applicability",
     # OCA/sales-workflow
     "sale_product_set_layout": "sale_product_set",
+    # OCA/pos
```
Comment (pedrobaeza) [inline openupgrade_scripts/apriori.py:78]: Please insert this in alphabetically order (of repos)

Comment (legalsylvain) [reply openupgrade_scripts/apriori.py:78]: The original file is not ordered correctly. I fixed it.

**Review by pedrobaeza — APPROVED** [review]

File: openupgrade_scripts/apriori.py (hunk 1 above)
Comment (FrancoMaxime) [reply openupgrade_scripts/apriori.py:78]: @legalsylvain Nice PR. Thanks for the work!

**Pull request merged**